packages/logger/src/index.ts

```
@@ -0,0 +1,87 @@
+ * 
+ * When SOURCEBOT_STRUCTURED_LOGGING_ENABLED=false (default):
+ * - Console output will be human-readable with colors
+ * - Logs will be formatted as: "timestamp level: [label] message"
+ */
+
+const { combine, colorize, timestamp, prettyPrint, errors, printf, label: labelFn, json } = format;
+
+const datadogFormat = format((info) => {
+    info.status = info.level.toLowerCase();
+    info.service = info.label;
+    delete info.label;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)